src/components/layout/sidebar/SideBar.module.css

```diff
@@ -93,11 +93,6 @@
     margin-bottom: 24px;
```

Comment:
Thw space between first and second sidebarItem seems to be too big. I prefer to keep the distance between sections uniform